src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -127,6 +126,19 @@ internal class MemberSignatureComparer : IEqualityComparer<Symbol>
             considerRefKindDifferences: true,
             typeComparison: TypeCompareKind.AllIgnoreOptions);
 
+        /// <summary>
+        /// This instance is used to determine if a partial method implementation matches the definition,
+        /// including differences ignored by the runtime.
+        /// </summary>
+        public static readonly MemberSignatureComparer PartialMethodsStrictComparer = new MemberSignatureComparer(
+            considerName: true,
+            considerExplicitlyImplementedInterfaces: true,
+            considerReturnType: true,
+            considerTypeConstraints: false,
+            considerCallingConvention: false,
+            considerRefKindDifferences: true,
+            typeComparison: TypeCompareKind.ConsiderEverything);
```

Comment:
>ConsiderEverything [](start = 44, length = 18)

I expected we'd ignore oblivious differences.

Reply:
Ah, yes, this should change. Sorry for overlooking this in my review 😓 The `DifferentReturnTypes_17` baseline should change as a result of this.

Reply:
Good catch, thanks.

---
In reply to: [490448310](https://github.com/dotnet/roslyn/pull/47576#discussion_r490448310) [](ancestors = 490448310)